plugins/builds/create.js

```diff
@@ -26,8 +27,11 @@ module.exports = (server) => ({
                 tokenGen: (buildId) =>
                     request.server.plugins.login.generateToken(buildId, ['build'])
             };
+            const config = hoek.applyToDefaults(payload, {
+                username: request.auth.credentials.username
```

Comment:
Hmmm why do we want to add them to the list? If they don't have permission then they shouldn't be creating the build... Right?